### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by mmck345** [SimonMarquis/FCM-toolbox#11]
Hi Simon
Great project as always..
Thanks for sharing the source code!
I encountered an error while building in the build.gradle file sync phase
`This version of the Android Support plugin for IntelliJ IDEA (or Android Studio) cannot open this project, please retry with version 2021.1.1 or newer.
`
I tried to find solutions to this error in Google, but they created other errors in the project.
Is there a way to build properly?

Thank you!

**Comment by SimonMarquis** [comment]
Hi @mmck345, as the error suggests, you might have to update to the latest Android Studio to build this project.

**Comment by mmck345** [comment]
Okay,
Thank you for your comment
I am using the most up to date version of Android Studio, what I am currently trying to do is take the resource folder and java code and manifest file and copy to a new project I have created I hope it will work and I will be able to build the project successfully.
Thank you!
![image](https://user-images.githubusercontent.com/52917429/149384251-5dafbba6-cc31-4993-b825-1f7eed503b0b.png)

**Comment by mmck345** [comment]
![image](https://user-images.githubusercontent.com/52917429/149392197-f7650598-02b6-436f-a019-947c02cc0301.png)


For anyone who needs this solution this has solved my problem
Thank you all!

**Comment by SimonMarquis** [comment]
Yeah, I'm currently using the Chimpmunk version (2021.2.1), though you should also be able to use the Bumblebee version (2021.1.1).
